src/det.jl

```diff
@@ -42,13 +42,24 @@ end
     -2*rem(s, 2) + 1
 end
 
+det(F::LU) = det(F.U) * _parity(F.p)
+
+function logdet(F::LU)
+    d, s = logabsdet(F.U)
+    d + log(s * _parity(F.p))
+end
+
+function logabsdet(F::LU)
+    d, s = logabsdet(F.U)
```

Comment:
What does this dispatch to?

Reply:
Good catch! It does to the default method in `LinearAlgebra` (like it did before). However, this is an opportunity for improvement, so I will do something nicer.